test/Microsoft.Azure.SignalR.Tests/ServiceLifetimeManagerFacts.cs

```diff
@@ -189,6 +189,68 @@ public async void ServiceLifetimeManagerOnlyBlazorHubProtocolTest(string functio
             Assert.Equal(1, (serviceConnectionManager.ServiceMessage as MulticastDataMessage).Payloads.Count);
         }
 
+        private sealed class TestServiceConnectionPrivate : TestServiceConnection
+        {
+            public ServiceMessage last = null;
+
+            public override Task WriteAsync(ServiceMessage serviceMessage)
+            {
+                last = serviceMessage;
+                return Task.CompletedTask;
+            }
+        }
+
+        [Fact]
+        public async void TestSendConnectionAsyncisOverwrittenWhenClientConnectionExisted()
+        {
+            var serviceConnectionManager = new TestServiceConnectionManager<TestHub>();
+            var clientConnectionManager = new ClientConnectionManager();
+
+            var context = new ClientConnectionContext(new OpenConnectionMessage("conn1", new Claim[] { }));
+            var connection = new TestServiceConnectionPrivate();
+            context.ServiceConnection = connection;
+            clientConnectionManager.AddClientConnection(context);
+
+            var manager = MockLifetimeManager(serviceConnectionManager, clientConnectionManager);
+
+            await manager.SendConnectionAsync("conn1", "foo", new object[] { 1, 2 });
+
+            Assert.NotNull(connection.last);
+            if (connection.last is MultiConnectionDataMessage m)
+            {
+                Assert.Equal("conn1", m.ConnectionList[0]);
+                Assert.Equal(1, m.Payloads.Count);
+                Assert.True(m.Payloads.ContainsKey(MockProtocol));

```

Comment:
>MockProtocol [](start = 51, length = 12)

payloads should not contain `blazor`

Reply:
Yes, it should.
Our resolver supports 3 different protocols (including a mocked protocol "blazor"),
and this mocked protocol has been designated in the lifetime manager creation options.